### PR TITLE
Updated API

### PR DESCRIPTION
**Pull request by uaArsen**
#43

### CODE REVIEW

**Comment by uaArsen** [comment]
#40

File: README.md [hunk 1]
```diff
@@ -13,10 +13,9 @@ Salesforce component is designed for Salesforce API integration.
 
 
 ### API version
-The component uses Salesforce - API Version 25.0, except:
+The component uses Salesforce - API Version 45.0, except:
 - Action: Lookup Object - API Version 39.0
```
Comment (ShkarupaNick) [inline README.md]: why Lookup Object wasn`t changed to v45?

File: spec/trigger.spec.js [hunk 2]
```diff
@@ -121,7 +121,7 @@ describe('Polling trigger test', function () {
           return connStub
         },
         execute: function (cfg, processResults) {
-          processResults(undefined, []);
+          // processResults(undefined, []);
```
Comment (ShkarupaNick) [inline spec/trigger.spec.js]: should it be removed?

Comment (uaArsen) [reply spec/trigger.spec.js]: removed

File: README.md (hunk 1 above)
Comment (uaArsen) [reply README.md]: Updated

**Pull request merged**